exercise 23/index.js

```diff
@@ -0,0 +1,7 @@
+let hour= prompt('enter the current hours');
+if(hour<0 || hour> 23)
+alert('input is incorrect');
+if( hour< 8 || hour >= 18)
+alert ('hour is outside of the office hours');
+if( hour>= 8 && hour< 18)
+alert('we are open');
```

Comment:
1. Need to convert prompt value into "number" type, before comparing it.
2. You are using many times `if(condition)`, but you can use `if(){} else if() {} else {}`.